WordPress/playground-tools
# Add Playground Tools issue templates

## What?
Adds issue template starting point for Playground Tools


## Why?
Creates guidelines for better error reporting

## How?
Creating an issue template

## Testing Instructions
1. [Allow issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) on the 
`Playground Tools` repo
2. Check or merge PR
3. Open a new issue and use the `Playground Tools issues - wp-now, VS Code extension, utilities` template
4. See if it suits the needs or needs updating
